src/Tools/AI Test Toolkit/src/TestSuite/AITLineStatus.Enum.al

```diff
@@ -0,0 +1,48 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+/// <summary>
+/// This enum has the Status of the AIT Line.
+/// </summary>
+enum 149031 "AIT Line Status"
+{
+    Extensible = false;
```

Comment:
Access = Public;

Reply:
It is default, no need polutting the code